src/js/sidebar/instructors/search.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Users search component
+ *
+ * @since [version]
+ * @version [version]
+ */
+
+// WP deps.
+import { __ } from '@wordpress/i18n';
+
+// Internal Deps.
+import { SearchUser } from '../../components';
+
+/**
+ * User search component for searching instructors.
+ *
+ * @since [version]
+ *
+ * @param {Object}   props
```

Comment:
```suggestion
```
is this intended for some doc rule I'm not aware about?

Reply:
Yea, when defining properties of an object like this JSDoc requires the main object itself to be defined (with or without a description) prior to the object's properties being defined below: https://jsdoc.app/tags-param.html#parameters-with-properties

elsint throws an error if the property is not defined